mix.exs

```diff
@@ -22,6 +27,55 @@ defmodule Wanda.MixProject do
     ]
   end
 
+  defp docs do
+    [
+      main: "readme",
+      logo: "priv/static/images/trento.svg",
+      extra_section: "GUIDES",
+      source_url: @source_url,
+      assets: "guides/assets/",
+      extras: extras(),
+      groups_for_extras: groups_for_extras(),
+      groups_for_modules: [
+        Domain: [~r/Trento.Domain.*/],
```

Comment:
This `groups_for_modules` is wrong hehe
Need to be updated to wanda

Reply:
updated